projects/guetzli/project.yaml

```diff
@@ -0,0 +1,4 @@
+homepage: "https://github.com/google/guetzli"
+primary_contact: "robryk@google.com"
+sanitizers:
```

Comment:
I'm still fixing some undefined behaviours (namely, shifts of negative integers).